apps/web/src/contexts/ABTestingContext/ABTestingContext.tsx

```diff
@@ -0,0 +1,40 @@
+import { FEATURE_FLAGS } from 'contexts/ABTestingContext/config'
+import { createContext, useCallback, useContext, useState } from 'react'
+
+interface ABTestingProviderProps {
+  children: React.ReactNode
+  ABTestUserResults: { [flag in FEATURE_FLAGS]: boolean }
+}
+
+type ABTestingContextType = {
+  featureFlagResults: { [flag in FEATURE_FLAGS]: boolean }
+  selectUserfeatureABResult: (flagKey: FEATURE_FLAGS) => boolean
+}
+
+const ABTestingContextManager = createContext({} as ABTestingContextType)
+
+function ABTestingManagerProvider({ ABTestUserResults, children }: ABTestingProviderProps) {
+  const [featureFlagResults] = useState<{ [flag in FEATURE_FLAGS]: boolean }>(ABTestUserResults)
+
+  const selectUserfeatureABResult = useCallback(
+    (flagKey: FEATURE_FLAGS) => {
+      if (typeof featureFlagResults[flagKey] === 'undefined') return false
+      return featureFlagResults[flagKey]
+    },
+    [featureFlagResults],
+  )
```

Comment:
suggest we remove this select function as it's not necessary. Once we get the context in the `useExperimentalFeatureEnabled` hook, we can use `useMemo` to derive the feature flag from the context and the provided feature name